client/src/App.js

```diff
@@ -15,11 +15,19 @@ function App() {
   const [members, setMembers] = useState(loadFromLocal('members') ?? []);
   const [isShown, setIsShown] = useState(false);
   const [remainingMembers, setRemainingMembers] = useState(members);
+  const [availableGroups, setAvailableGroups] = useState(
+    loadFromLocal('groups') ?? []
+  );
+  const [cantDeleteGroup, setCantDeleteGroup] = useState('');
```

Comment:
I was put off a bit by the naming. Maybe you can think again to find a name that is more precise. 
What do you need this additional state for?